save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/validation/ValidationUtils.kt

```diff
@@ -0,0 +1,40 @@
+/**
+ * Module that implements different types of validation.
+ */
+
+package com.saveourtool.save.validation
+
+/**
+ * Default amount of characters allowed for names
+ */
+const val NAMING_ALLOWED_LENGTH = 64
+
+/**
+ * Check if name is valid.
```

Comment:
Name for which entities?

Reply:
For now just for three main entities: `Project`, `Organization` and `Contest`, but generally I guess we will have the same requirements for any names